it/it-tests/src/test/java/it/serverSystem/RestartTest.java

```diff
@@ -47,22 +52,29 @@ public void stop() {
     }
   }
 
-  /**
-   * SONAR-4843
-   */
   @Test
-  public void restart_forbidden_if_not_dev_mode() throws Exception {
+  public void restart_in_prod_mode_requires_admin_privileges_and_restarts_WebServer_and_ES() throws Exception {
     // server classloader locks Jar files on Windows
     if (!SystemUtils.IS_OS_WINDOWS) {
       orchestrator = Orchestrator.builderEnv()
+        .setOrchestratorProperty("orchestrator.keepWorkspace", "true")
         .build();
       orchestrator.start();
+
       try {
-        orchestrator.getServer().adminWsClient().systemClient().restart();
+        orchestrator.getServer().wsClient().systemClient().restart();
```

Comment:
- example of use in IT: https://github.com/SonarSource/sonarqube/blob/master/it/it-tests/src/test/java/it/component/ComponentsWsTest.java#L42
- you need a new method system() in https://github.com/SonarSource/sonarqube/blob/master/sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java#L43
- you need to map with specific WsRequest
- I should have warn you earlier about it